charts/internal/seed-controlplane/charts/alicloud-cloud-controller-manager/templates/configmap-observability.yaml

```diff
@@ -1,49 +0,0 @@
-{{- if not .Values.gep19Monitoring }}
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cloud-controller-manager-observability-config
```

Comment:
Not introduced with this PR but we could delete https://github.com/gardener/gardener-extension-provider-alicloud/blob/9b5fad716d0ae4abb7a2e2ba6c7733b5ec5fb414/pkg/controller/controlplane/valuesprovider.go#L113 and https://github.com/gardener/gardener-extension-provider-alicloud/blob/9b5fad716d0ae4abb7a2e2ba6c7733b5ec5fb414/pkg/controller/controlplane/valuesprovider.go#L269-L272.
Introduced with https://github.com/gardener/gardener-extension-provider-alicloud/pull/720.